src/app/components/User/User.tsx

```diff
@@ -1,4 +1,4 @@
-import { useCurrentUser } from '../../hooks/user';
+import { useCurrentUser } from '../../hooks/useUser';
```

Comment:
To have a similar code style like other hooks, I would rename it
```suggestion
import useCurrentUser from '../../hooks/useCurrentUser';
```

Reply:
thanks, solved